### ISSUE TITLE
[BUG] Named Repeater Blocks gets mixed from other entries

### ISSUE THREAD

**Issue by zomars** [bolt/bolt#6305]
When I create a new entry with named repeaters, it appears pre-filled with blocks from other previously saved entries (I'm not duplicating them). When I remove them I get a console error:

```
Uncaught TypeError: b._setCount is not a function
    at HTMLButtonElement.<anonymous> (bolt.js?44e7fa8448:9)
    at HTMLFieldSetElement.dispatch (lib.js?ef1ee19aeb:25)
    at HTMLFieldSetElement.q.handle (lib.js?ef1ee19aeb:24)
```

Details
-------

 - Relevant Bolt Version: Bolt 3.5.0 alpha 0
 - Install type: Composer install
 - PHP version: 7.0
 - Used web server: Apache 2.4

**Comment by eduardomart** [comment]
I can confirm it. It also happens when you create subsequent records, the deleted Named Repeater gets automatically added to them.

I'll mention @rossriley since this is his upcoming feature.

**Comment by eduardomart** [comment]
Also, the Named Repeater area dissapears, and seems to cause some issues with the overall backend template. See here: http://take.ms/1pi8H

**Comment by bobdenotter** [comment]
Fixed by #6314
